assets/js/admin.js

```diff
@@ -1,6 +1,5 @@
-jQuery( function($){
-
-	"use strict";
+jQuery( function ( $ ) {
```

Comment:
You can ignore this file, since I just run the formatter for JS and it was updated.